### PR TITLE
✨ add spacer

### PR DESCRIPTION
**Pull request by YounesBessa**

### CODE REVIEW

File: examples/next-typescript/pages/spacer.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  return (
+    <>
+      <Container
+        gap={10}
+        style={{
+          maxWidth: 700,
+          margin: "0 auto",
+        }}
+      >
+        <p>vertical</p>
```
Comment (AntoineKM) [inline examples/next-typescript/pages/spacer.tsx:16]: should be a `Text`component

File: examples/next-typescript/pages/spacer.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          <Container style={{ background: "white" }} />
+          <Spacer x={23} y={213} />
+          <Container style={{ background: "white" }} />
+        </Container>
+      </Container>
+    </>
+  );
+};
+
+const SpacerContainer = styled(Container)`
```
Comment (AntoineKM) [inline examples/next-typescript/pages/spacer.tsx:42]: don't need it, you can set a gap to your `Container`

File: examples/next-typescript/pages/spacer.tsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  margin-top: 24px;
+`;
+
```
Comment (AntoineKM) [inline examples/next-typescript/pages/spacer.tsx:45]: make a styled component of spacer with a background-color to see the size it takes

File: packages/kitchen/src/components/Spacer/index.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+const Spacer = styled(({ ...props }: SpacerProps) => {
+  return <span {...props}></span>;
+})<SpacerProps>`
+  display: block;
+  width: 1px;
+  height: 1px;
+  min-width: 1px;
+  min-height: 1px;
+  // horizontal margin can be set with x and it can either be a number or a key from the theme
```
Comment (AntoineKM) [inline packages/kitchen/src/components/Spacer/index.tsx:19]: remove comments

File: packages/kitchen/src/components/Spacer/index.tsx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  margin-left: ${(props) =>
```
Comment (AntoineKM) [inline packages/kitchen/src/components/Spacer/index.tsx:20]: to write less you could destructure it

**Review by AntoineKM — CHANGES_REQUESTED** [review]

File: packages/kitchen/src/components/Spacer/index.tsx (hunk 5 above)
Comment (YounesBessa) [reply packages/kitchen/src/components/Spacer/index.tsx:20]: what do you mean?

Comment (AntoineKM) [reply packages/kitchen/src/components/Spacer/index.tsx:19]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

**Review by jerembdn — APPROVED** [review]

**Review by AntoineKM — APPROVED** [review]

**Pull request merged**